frontend/src/composables/use-analytics.ts

```diff
@@ -73,7 +60,7 @@ export const useAnalytics = () => {
       props: {
         ...isomorphicProps.value,
         ...windowProps.value,
-        ...payload, // can override mandatory props
+        ...payload, // will override matching mandatory props
```

Comment:
It's actually not true now that the ESLint rule disallows it :slightly_smiling_face: 

Speaking of which, you'll need to remove these prop names from the ESLint rule. Or should we keep them/change the rule to infer from the name that we're accidentally duplicating built-in event data?

Reply:
I've removed the comment and the prop names from the Eslint rule for now, thanks!

The separate issue sounds like a welcome addition, a "you might be duplicating a built-in prop" notification. I wonder if name inference would be reliable enough to be useful? Like if someone tried to add a prop called `userAgent`, for example, that wouldn't match `ua`.

Reply:
Issue here: https://github.com/WordPress/openverse/issues/2597